### DISCUSSION TITLE
Chats didn't mount only with internal parser in IPED 3.18.8

### DISCUSSION THREAD

**Discussion by Klarissa-ksj** [sepinf-inc/IPED#757] [Q&A]
Hello,

I processed a Xiaomi cell phone with "phoneParsersToUse = internal" in conf/AdvancedConfig.txt and it didn't mount the whatsapp chats. Then I changed the parameter to "external" (UFED) and it suceeded to mount the chats.

Is there any problem with this type of cellphone? I often use  phoneParsersToUse = internal in default\conf\advancedconfig.txt.

The logs go 
[IPED-SearchApp.log](https://github.com/sepinf-inc/IPED/files/7164813/IPED-SearchApp.log)
[IPED-SearchAppwithoutWhatsAppChats_internal.log](https://github.com/sepinf-inc/IPED/files/7164819/IPED-SearchAppwithoutWhatsAppChats_internal.log)
attached.

**Comment by lfcnassif** [comment]
Maybe the database schema could have changed in newer whatsapp versions, I think it is not related with the cellphone manufacturer. Or maybe it could be a bug. We would need the processing log, not the analysis log, to investigate. And If you could share the msgstore.db file privately, that should help a lot.

**Reply by lfcnassif** [reply]
This was resolved in #758.
